packages/actions/scripts/prepare-release.mjs

```diff
@@ -14,6 +14,9 @@ async function main() {
     const packageJson = JSON.parse(await fs.readFile('package.json', 'utf-8'));
     const apiVersion = packageJson.apiVersion;
     const sdkVersion = packageJson.version;
+    const prId = process.env.CI_PR_ID;
```

Comment:
What is this for?

Reply:
To have these files available on a per PR basis. Right now these files are overwritten on each SDK PR build 😕